lib/ansible/modules/web_infrastructure/jenkins_script.py

```diff
@@ -154,6 +159,7 @@ def main():
             validate_certs=dict(required=False, type="bool", default=True),
             user=dict(required=False, no_log=True, type="str", default=None),
             password=dict(required=False, no_log=True, type="str", default=None),
+            timeout=dict(required=False, no_log=True, type="int", default=10),
```

Comment:
Timeout don't require `no_log`. Option is for hiding sensitive information like auth_token or password from log_file.